internal/commands/pull/command.go

```diff
@@ -168,30 +197,35 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 	ui.Print(terminal.NewTextLog("Saved app to disk"))
 
-	if cmd.inputs.IncludeDependencies {
+	if cmd.inputs.IncludePackageJSON || cmd.inputs.IncludeNodeModules {
+		logStr := "as a package.json file"
+		if cmd.inputs.IncludeNodeModules {
+			logStr = "as a node_modules .zip"
```

Comment:
since we don't quite know its actually a `.zip`
```suggestion
			logStr = "as a node_modules archive"
```